ops/manifests/manifest.py

```diff
@@ -124,13 +60,15 @@ class Manifests:
     def __init__(
         self,
         name: str,
+        model_name: str,
         app_name: str,
         base_path: PathLike,
         manipulations: List[Manipulation] = None,
     ):
         """Create Manifests object.
 
         @param name:         Uniquely idenitifes these released manifests.
+        @param model_name:     Charm model name
```

Comment:
```suggestion
        @param model_name:   Charm model name
```

Reply:
hmm, i'll have to go modify the importers of this

Reply:
it looks like initializing a `Manifests`, could required all sorts of NEW params added:


a `Manifests` has
* `name` -- what does the charm call this blob of manifests
* `base_path` -- where are these manifests on the file system
* `manipulations` -- what changes does the charm want to make to them
* `environment` -- maybe this should include things like
    * charm app name
    * charm model name
    * other things we haven't thought of yet

Rather than adding `model-name` into the args list, i think i'd rather see some class hold which holds the environment data, maybe even something that could easily be constructed naturally from `CharmBase`?

Is this too much abstraction?